src/components/App/App.jsx

```diff
@@ -1,13 +1,13 @@
-import React, { useState, useEffect } from 'react';
-import BmiForm from './BmiForm';
-import Info from './Info';
-import Bar from './Bar';
-import './App.css';
-import 'materialize-css/dist/css/materialize.min.css';
-const uuidv1 = require('uuid/v4');
+import React, { useState, useEffect } from "react";
+import BmiForm from "../BmiForm/BmiForm";
+import Info from "../Info/Info";
+import Bar from "../Bar/Bar";
+import "./App.css";
+import "materialize-css/dist/css/materialize.min.css";
+const uuidv1 = require("uuid/v4");
```

Comment:
Can you replace all these double quotes with single quotes?

Reply:
They're not in the same folder anymore

Reply:
ohh sorry, i just got it wrong